launch/client.py

```diff
@@ -2893,6 +2907,39 @@ def completions_sync(
             resp = json.loads(response.response.data)
         return resp
 
+    def completions_stream(
+        self,
+        endpoint_name: str,
+        prompt: str,
+        max_new_tokens: int,
+        temperature: float,
+    ) -> Iterable[CompletionStreamV1Response]:
+        """
+        Run prompt completion on an LLM endpoint in streaming fashion. Will fail if endpoint does not support streaming.
+
+        Parameters:
+            endpoint_name: The name of the LLM endpoint to make the request to
+
+            prompt: The prompt to send to the endpoint
+
+            max_new_tokens: The maximum number of tokens to generate for each prompt
+
+            temperature: The temperature to use for sampling
+
+        Returns:
+            Iterable responses for prompt completion
+        """
+        request = {"max_new_tokens": max_new_tokens, "prompt": prompt, "temperature": temperature}
+        response = requests.post(
+            url=f"{self.configuration.host}/v1/llm/completions-stream?model_endpoint_name={endpoint_name}",
```

Comment:
Hmm not using the autogen?

I guess it's not the end of the world since we've decided to go hand-written for the other client.

Maybe not worth worry about for now.

Reply:
didn't have time to figure out streaming for autogen